youtube_dl/extractor/niconico.py

```diff
@@ -395,7 +414,9 @@ def get_video_info(items):
 
         webpage_url = get_video_info('watch_url') or url
 
-        owner = api_data.get('owner', {})
+        # Note: cannot use api_data.get('owner', {}) because owner may be set to "null"
+        # in the JSON, which will cause None to be returned instead of {}.
+        owner = api_data.get('owner') or {}
```

Comment:
This won't help `owner` is set to `True`-ish non dict.

Reply:
I don't see why that would practically happen though. I imagine that they will only return two things:
- an object if a owner is present
- null if an owner is not present

This is the pattern used in other extractors too, for example https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/instagram.py#L274.

What's your suggestion?

Reply:
You can't make any assumptions on data returned. This pattern is flawed. Suggestion is to make it work in all possible cases.